### DISCUSSION TITLE
Is there a reason the code injector can't be cached in a field, instead of being instantiated every time?

### DISCUSSION THREAD

**Discussion by vsajip** [javacc21/javacc21#210] [Q&A] [answered]
https://github.com/javacc21/javacc21/blob/933ce5dcc3f8caf82aa63d7e05f144394a867c19/src/java/com/javacc/Grammar.java#L440-L444

**Comment by revusky** [comment, marked as answer]
Offhand, I don't see any reason, no. Try it I guess.
